### PR TITLE
Add support for ActiveMQ 5.14.0

### PR DESCRIPTION
**Pull request by iemejia**

### CODE REVIEW

File: 5.14.0/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ENV ACTIVEMQ apache-activemq-$ACTIVEMQ_VERSION
+ENV ACTIVEMQ_TCP=61616 ACTIVEMQ_AMQP=5672 ACTIVEMQ_STOMP=61613 ACTIVEMQ_MQTT=1883 ACTIVEMQ_WS=61614 ACTIVEMQ_UI=8161
+
+ENV ACTIVEMQ_HOME /opt/activemq
+
+RUN set -x && \
+    curl -s https://archive.apache.org/dist/activemq/$ACTIVEMQ_VERSION/$ACTIVEMQ-bin.tar.gz | tar xvz -C /opt && \
+    ln -s /opt/$ACTIVEMQ $ACTIVEMQ_HOME && \
+    useradd -r -M -d $ACTIVEMQ_HOME activemq && \
+    chown activemq:activemq /opt/$ACTIVEMQ -R
```
Comment (rmohr) [inline 5.14.0/Dockerfile]: could you add the chown change for the symlink too?

File: 5.14.0/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,20 @@
+FROM java:8-jre
+
+ENV ACTIVEMQ_VERSION 5.14.0
+ENV ACTIVEMQ apache-activemq-$ACTIVEMQ_VERSION
+ENV ACTIVEMQ_TCP=61616 ACTIVEMQ_AMQP=5672 ACTIVEMQ_STOMP=61613 ACTIVEMQ_MQTT=1883 ACTIVEMQ_WS=61614 ACTIVEMQ_UI=8161
+
+ENV ACTIVEMQ_HOME /opt/activemq
+
+RUN set -x && \
+    curl -s https://archive.apache.org/dist/activemq/$ACTIVEMQ_VERSION/$ACTIVEMQ-bin.tar.gz | tar xvz -C /opt && \
```
Comment (rmohr) [inline 5.14.0/Dockerfile]: Could you also add the -S flag?

**Comment by rmohr** [comment]
Thanks, only one last thing, could you squash these two commits into one?

**Comment by iemejia** [comment]
Rebased as you asked, it should be ok now.

**Pull request merged**